packages/application-generic/src/usecases/add-job/merge-or-create-digest.usecase.ts

```diff
@@ -50,7 +50,8 @@ export class MergeOrCreateDigest {
   ): Promise<MergeOrCreateDigestResultType> {
     const { job } = command;
 
-    const digestMeta = job.digest as IDigestBaseMetadata | undefined;
+    const digestMeta =
+      command.chimeraData ?? (job.digest as IDigestBaseMetadata | undefined);
```

Comment:
Adding the `chimeraData` to the digestMeta as the first option, falling back to the non-Echo data.